htcondor.yml

```diff
@@ -294,11 +294,11 @@
       when: htcondor_role_submit
     - grycap.htcondor
     - name: usegalaxy-eu.htcondor_release
-      when: htcondor_role_submit
+      when: htcondor_role_submit and inventory_hostname != "maintenance.galaxyproject.eu"
```

Comment:
I guess this will be removed again once the nspawn container stops doing that?
(just for my better understanding)

Reply:
The problem here is that `usegalaxy-eu.htcondor_release` is designed to run on max one submitter node (this is of course a bad solution and needs to be addressed later, but we want to have the monitoring working properly as well). If we do not prevent `usegalaxy-eu.htcondor_release` from running on more than one submitter node, then #896 comes back.

So unfortunately no, this will not be removed again once the nspawn container can be decommissioned. It will not be as simple as that :(